R/ggplotly.R

```diff
@@ -168,6 +170,9 @@ gg2list <- function(p){
     gglayout <- built$panel$layout
     ## invert rows so that plotly and ggplot2 show panels in the same order
     gglayout$plotly.row <- max(gglayout$ROW) - gglayout$ROW + 1
+    ## ugh, ggplot counts panel right-to-left & top-to-bottom
+    ## plotly count them right-to-left & *bottom-to-top*
```

Comment:
Please use a single `#` for comments (see https://github.com/ropensci/plotly/wiki/Development-guidelines#coding-style and https://google-styleguide.googlecode.com/svn/trunk/Rguide.xml#comments).